Mage/src/main/java/mage/abilities/effects/ContinuousEffectImpl.java

```diff
@@ -301,9 +301,6 @@ protected static boolean isCanKill(DynamicValue toughness) {
             // count this class as used for "-{something_positive}"
             return true;
         }
-        if (toughness instanceof DomainValue) {
-            return ((DomainValue) toughness).getAmount() < 0;
-        }
```

Comment:
?

Reply:
Should this be `.calculate()` instead of .`getAmount()`?

Reply:
These removed lines were vestigial code that didn't do anything. The method `DomainValue.getAmount` always returned a hardcoded 1 (and this is the only place it was called).

Reply:
I get you.

So have all of the classes which use the multiply with a negative number been switched to a sign inversion so they get caught by the if-statement above?

Reply:
I'm going to try to improve this method so it's more generic and not hard coded to specific `DynamicValue` classes.

Reply:
As part of the PR, or another?

Reply:
My plan is to add a new method to the `DynamicValue` interface with a default that only a few classes will need to override. It should be a small enough amount of code to do within this PR.